fenix/app/metrics.yaml

```diff
@@ -9429,6 +9429,23 @@ private_browsing_shortcut_cfr:
       - android-probes@mozilla.com
     expires: 122
 
+server_knobs:
+  validation:
+    disabled: true
+    type: event
+    description: |
+      Temporary metric recorded at the same time as
+      "tabs_tray.new_tab_tapped" to validate that the Glean Server Knobs
+      functionality is working correctly.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1823682
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1823682#c2
+    notification_emails:
+      - android-probes@mozilla.com
+      - brosa@mozilla.com
+    expires: 116
```

Comment:
@rosahbruno please note that this will expire in 2 releases.  If you want to extend it please do it before June 1st or this will be removed in 116.  Thanks